Resources/doc/installtion.rst

```diff
@@ -0,0 +1,83 @@
+Installation
+============
+
+This chapter assumes you have Composer installed globally, as explained
+in the `installation chapter`_ of the Composer documentation.
+
+.. note::
+
+   The ODM requires the `MongoDB driver`_ (``mongodb``).
+
+Install the bundle with Symfony Flex
+------------------------------------
+
+A Flex recipe for the DoctrineMongoDBBundle is provided through a Contrib Recipe
+therefore you need to allow its usage:
+
+.. code-block:: bash
+
+    composer config extra.symfony.allow-contrib true
```

Comment:
Not sure if this is needed anymore because Flex now asks you if you want to install a contrib package (always, just this time, only for this vendor, etc.)